modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/model/listener/CompanyModelListener.java

```diff
@@ -29,11 +37,39 @@ public class CompanyModelListener extends BaseModelListener<Company> {
 
 	@Override
 	public void onAfterCreate(Company company) throws ModelListenerException {
+		_processDeprecationFeatureFlags(company.getCompanyId());
+	}
+
+	@Activate
+	protected void activate() {
+		_companyLocalService.forEachCompanyId(
```

Comment:
@thiago-buarqque Method invoked in activate method do not need TransactionCommitCallback as the company is already created.

Reply:
Fixed it, thank you